tests/test_typing/bad/err6.ml

```diff
@@ -5,124 +5,213 @@ open Runtime_ocaml.Runtime
 [@@@ocaml.warning "-4-26-27-32-41-42"]
 
 module Enum = struct
- type t =
-    | Int of integer
-    | Dec of decimal
-    | Dat of date
-  end
+  type t = Int of integer | Dec of decimal | Dat of date
```

Comment:
Huh, is this file expected to be committed in ?
`err6.catala_en` is supposed to just have a type error and no output file (which should be inlined anyway)